internal/src/main/java/no/unit/nva/model/internal/ResourceDao.java

```diff
@@ -0,0 +1,23 @@
+package no.unit.nva.model.internal;
```

Comment:
The DAOs may and (probably will) contain information that is connected to the access patterns of the database. We can start developing it here, but it is likely that it will be moved to the publication-api